beta/adobe-journey-optimizer/push-messaging-guide.md

```diff
@@ -0,0 +1,81 @@
+
+#### Adobe Launch
+
+[Adobe Launch or Data Collection](https://experienceleague.adobe.com/docs/launch/using/home.html) is the next generation of mobile SDK management capabilities which enables data flow from mobile SDK to Experience Platform. It provides capabilities to register extensions, create rules and data elements to send data from your mobile app to the Adobe experience cloud solutions. With respect to the push notifications data flow, the primary configurations required in Launch are :
+
+- Creating a datastream to configure the profile and experience event datasets against which the data flows into experience platform. 
+- Creating a client-side mobile property and adding extensions. Mobile SDK closely integrates with these extensions to provide a seamless data collection experience.
+- Registering the mobile app bundle identifier and app credentials that would help to uniquely identify and validate sanity of the app at the time of delivering push notification. 
+
+You will find detailed steps for these configurations [here](https://aep-sdks.gitbook.io/docs/beta/adobe-journey-optimizer).
+
+#### Real-time Customer Profile
+
+[Real-time Customer Profile](https://experienceleague.adobe.com/docs/experience-platform/profile/home.html) is the component in [Experience Platform](https://experienceleague.adobe.com/docs/experience-platform.html) that allows you to see a holistic view of each individual customer by combining data from multiple channels, including web, mobile, CRM, and third party. Profile allows you to consolidate your customer data into a unified view offering an actionable, timestamped account of every customer interaction. 
+
+Static data that identifies the user of the mobile app such as a push token, is stored against the user's profile as record data while the interactions the user does with push notifications are tracked as time-series events data. 
+
+#### Adobe Journey Optimizer
+
+Once your mobile app integrations with above mentioned components are in place and your customer profiles are available as Real-time customer profiles, you can take advantage of powerful audience segmentation capabilities in [Adobe Journey Optimizer](https://experienceleague.adobe.com/docs/journey-optimizer/using/ajo-home.html) to ensure the optimal experience for each person.
+
+
+
+### Push Data Flow Steps
+
+Refer to the data flow diagram to identify what each of these bullets represent :
+
+1. The customer develops a mobile app on Android or iOS that they would release to their users. In order to use the push capability provided by push providers i.e APNS by Apple and FCM by Google, the mobile app registers itself and enables the push capability.
```

Comment:
"In order to use the push capability provided by the providers (APNS for Apple and FCM for Google), the mobile app..."